src/rowtransforms.jl

```diff
@@ -0,0 +1,60 @@
+struct NormalizeRow <: DataAugmentation.Transform
+	normstats
+	normcols
+end
+
+struct Categorify <: DataAugmentation.Transform
+	pooldict
+	categorycols
+end
+
+struct FillMissing <: DataAugmentation.Transform
+	fmvals
+	contcols
+	catcols
+end
```

Comment:
```suggestion
struct FillMissing{T, S} <: DataAugmentation.Transform
	fmvals::T
	contcols::S
end
```

Reply:
The holy grail would be to avoid any intermediate heap allocations here. I'm not sure if that's possible though, so perhaps it's better to create a vector anyhow.